src/Ads/AdsRecommendationsService.php

```diff
@@ -156,6 +205,45 @@ public function get_google_recommendations( $args ): array {
 				$campaign = $row->getCampaign();
 				$customer = $row->getCustomer();
 
+				$recommendation_type    = RecommendationType::name( $recommendation->getType() );
```

Comment:
Is there any specific intention to call these here? Since we already have $types available, could we use that instead of making calls to the service class?